packages/kbn-test-eui-helpers/src/rtl_helpers.tsx

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { screen, within } from '@testing-library/react';
+
+export const getButtonGroupInputValue = (testId: string) => () => {
+  const buttonGroup = screen.getByTestId(testId);
+  return within(buttonGroup).getByRole('button', { pressed: true });
```

Comment:
Interesting. So `checked` didn't work, but `pressed` does.

Reply:
It actually did work, but the implementation from eui side changed and it stopped - they had radio inputs before and now these are buttons so I had to update it. This helper is so simple it's questionable it makes sense to have it here but I think it's a good opportunity to start this long planned package 😁